crates/store/re_video/src/decode/ffmpeg.rs

```diff
@@ -521,15 +590,12 @@ fn read_ffmpeg_output(
                     frame_info.presentation_timestamp
                 );
 
-                debug_assert_eq!(pix_fmt, "rgb24");
-                debug_assert_eq!(width as usize * height as usize * 3, data.len());
```

Comment:
it would be nice to still check something like

`sebug_assert_eq!(data.len() * 8, width * height  * pixel_format.bits_per_pixel());`